src/main/java/com/desiato/puresynth/services/AuthenticationService.java

```diff
@@ -58,7 +59,7 @@ public boolean isUserAuthenticated(PureSynthToken pureSynthToken) {
         Optional<Session> sessionOpt = sessionRepository.findById(pureSynthToken.value());
 
         if (sessionOpt.isEmpty()) {
-            logger.warn("Token authentication failed: Session not found for token {}", pureSynthToken);
+            logger.warn("Token authentication failed: Session not found for token {}");
```

Comment:
You have left interpolation here `{}` without any param.

Reply:
what parameter I should put there?

Reply:
You had:
`logger.warn("Token authentication failed: Session not found for token {}", pureSynthToken);`

Now you removed `pureSynthToken` but you left `{}` without any parameter for interpolation.

Reply:
I did not understood initially what you meant. Now I also removed the ```{}```